Signal/src/call/CallService.swift

```diff
@@ -75,6 +75,7 @@ enum CallError: Error {
     case disconnected
     case externalError(underlyingError: Error)
     case timeout(description: String)
+    case unexpected(description: String)
```

Comment:
I'm a little wary of how we're using assertionError, since they're used in guards, not assertion.  Maybe this should be guardError?  Anyway, I've switched to assertionError until we come up with something better.